.github/workflows/production-heartbeat.yml

```diff
@@ -9,17 +9,9 @@ on:
     # Days were chosen to run only from Monday through Friday.
     - cron: '45 13,17,21 * * 1,2,3,4,5'
 jobs:
-  production-heartbeat:
-    runs-on: macos-latest
-    steps:
-      - name: Invoke v4 workflow
-        uses: actions/github-script@v6
-        with:
-          github-token: ${{ secrets.SVC_CLI_BOT_GITHUB_TOKEN }}
-          script: |
-            await github.rest.actions.createWorkflowDispatch({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              workflow_id: 'production-heartbeat.yml',
-              ref: 'dev-4'
-            });
+  smoke-test:
```

Comment:
This looks incorrect to me. I'm not seeing where the logic that would call out to PagerDuty is?

Reply:
Ah, good point! I don't think I connected that the purpose of that was to callout to PagerDuty. Is that what the createWorkflowDispatch is doing here?

Reply:
@jfeingold35 here's the successful run to the heartbeat! Thanks for walking through that together. There might be some additional improvements to be made here too (possible follow-up WI).

https://github.com/randi274/sfdx-scanner/actions/runs/14801505744/job/41561089789

I did notice in the output of the job that we don't have python installed either so I made a follow-up WI for that:  https://github.com/randi274/sfdx-scanner/actions/runs/14801505744/artifacts/3052802686